tests/playwright/src/utility/operations.ts

```diff
@@ -454,3 +454,35 @@ export async function setStatusBarProvidersFeature(
   const experimentalPage = await settingsBar.openTabPage(ExperimentalPage);
   await experimentalPage.setExperimentalCheckbox(experimentalPage.statusBarProvidersCheckbox, enable);
 }
+
+export async function readFileInVolumeFromCLI(volumeName: string, fileName: string): Promise<string> {
+  return test.step('Read file in volume from CLI', async () => {
+    try {
+      let command: string;
+
+      if (isMac || isWindows) {
+        // macOS and Windows: Use podman machine to SSH into the VM
+        const pathInVM = `/var/lib/containers/storage/volumes/${volumeName}/_data/${fileName}`;
+        command = `podman machine ssh cat ${pathInVM}`;
+      } else if (isLinux) {
+        // Linux: Assume Podman is running natively
+        // Detect if running rootless by checking env or fallback to rootful
+        const isRootless = process.getuid && process.getuid() !== 0;
+        const basePath = isRootless
+          ? `${process.env.HOME}/.local/share/containers/storage/volumes`
```

Comment:
I would suggest to rely on nodejs handling home dir (`os.homedir()`) rather than on env. vars.